test/integration/controllers/get-list.js

```diff
@@ -81,6 +87,20 @@ describe('Integration Tests', function() {
                 });
             });
 
+            it('should get all user records with populated data', function(done) {
+                request(app.getExpressApplication()).
+                get('/users').
+                set('Content-Type', 'application/json').
+                expect(200).
+                end(function(err, res) {
+                    should.not.exist(err);
+
```

Comment:
Lets also assert against the user so it's clear who the user / company populated pair is.

Also, this test is failing https://travis-ci.org/crimsonronin/express-json-api/jobs/103050879 but it is due to the data import process, not the test. I think we will have to modify the `db.import` function, which should be use synchronous promises, however it doesn't seem to be working.

I can attempt to fix the import function unless you want to have a go at it @MrBaptastic ?